projects/layout/src/modules/box/box-controls.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'sky-box-controls',
+  templateUrl: './box-controls.component.html',
+})
+export class SkyBoxControlsComponent {
```

Comment:
Do we need a JSDocs comment to explain what the "controls" component does?

Reply:
Yes! That was a total miss... I'll get that going.

Reply:
Done.